src/Analyzers/Async/src/Microsoft.DotNet.Analyzers.Async.csproj

```diff
@@ -3,10 +3,11 @@
   <PropertyGroup>
     <Description>Analyzer package for async best practices.</Description>
     <VersionPrefix>$(ExperimentalVersionPrefix)</VersionPrefix>
-    <VerifyVersion>false</VerifyVersion>
     <TargetFramework>netstandard1.3</TargetFramework>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <BuildOutputTargetFolder>analyzers/dotnet/cs/</BuildOutputTargetFolder>
+      <!-- Analyzers cannot be referenced as a ProjectReference. -->
```

Comment:
You've got a `<ProjectReference />` from `test/Microsoft.DotNet.Analyzers.Async.Tests.csproj` to this project. Did you mean `as a Reference`?

Reply:
This is deliberate. Analyzers, like MSBuild tasks, are "special package" types. Although they carry assemblies, they don't end up in the `lib/` folder of a NuGet package. As a result, PackageRef and ProjectRef are not interchangeable, so I've made it an explicit ProjectRef.

Reply:
I'll update the comment to clarify.

Reply:
I understand the context but saying "PackageReference" here confuses the issue. My thought was just to say "as a Reference".

Reply:
I updated the comment. Let me know if the update is clearer.